### PR TITLE
edited .drone.yml to reflect the move of images from dockerhub to quay

### PR DESCRIPTION
**Pull request by alster-wasser**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
Images have been moved to quay; this PR edits the pipeline to reflect that

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```NONE
```

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -153,7 +153,7 @@ pipeline:
       --set=kubermatic.controller.image.tag=${DRONE_COMMIT_SHA}
       --set=kubermatic.api.image.tag=${DRONE_COMMIT_SHA}
       --set=kubermatic.rbac.image.tag=${DRONE_COMMIT_SHA}
-    image: kubeciio/helm
```
Comment (mrIncompetent) [inline .drone.yml]: This is a drone plugin - which we should not move. We don't own the image

File: .drone.yml [hunk 2]
```diff
@@ -220,7 +220,7 @@ pipeline:
...
+    image: quay.io/kubermatic/helm:2.11.0-2
```
Comment (mrIncompetent) [inline .drone.yml]: https://github.com/kubermatic/kubermatic/pull/2486/files#r239407511

File: .drone.yml [hunk 3]
```diff
@@ -270,7 +270,7 @@ pipeline:
...
```
Comment (mrIncompetent) [inline .drone.yml]: https://github.com/kubermatic/kubermatic/pull/2486/files#r239407511

File: .drone.yml [hunk 4]
```diff
@@ -336,7 +336,7 @@ pipeline:
...
```
Comment (mrIncompetent) [inline .drone.yml]: https://github.com/kubermatic/kubermatic/pull/2486/files#r239407511

File: .drone.yml (hunk 2 above)
Comment (alster-wasser) [reply .drone.yml]: will put kubeciio/helm back in.

**Comment by mrIncompetent** [comment]
Force push after a rebase

**Comment by mrIncompetent** [comment]
@lindtdev This PR can be closed?

**Pull request closed without merging**